asteroid/masknn/attention.py

```diff
@@ -0,0 +1,229 @@
+import torch.nn.functional as F
+import torch.nn as nn
+from torch.nn.modules.activation import MultiheadAttention
+from asteroid.masknn import activations, norms
+import torch
+from asteroid.utils import has_arg
+
+class ImprovedTransformedLayer(nn.Module):
+    def __init__(self, d_model, n_heads, dim_ff, dropout=0., activation="relu", bidirectional=True, norm="gLN"):
```

Comment:
What is the accepted name for the  in other packages (torch, huggingface, fairseq, espnet)? I'd like to have the same name so that it's easier to enter Asteroid's source code.
I saw `embed_dim` in torch, `n_feats` in espnet now.